### PR TITLE
media-libs/libass: Fix building for x32, bug #509460

### PR DESCRIPTION
**Pull request by bjacquin**
Package-Manager: portage-2.2.20.1

### CODE REVIEW

**Comment by mgorny** [comment]
@gentoo/video

File: media-libs/libass/libass-0.13.0.ebuild [hunk 1]
```diff
@@ -32,7 +32,16 @@ done
 DOCS="Changelog"
 
 multilib_src_configure() {
+	local myconf
+
+	if has ${MULTILIB_ABI_FLAG} ${ASM_MULTILIB_ABIS} ; then
```
Comment (luke-jr) [inline media-libs/libass/libass-0.13.0.ebuild]: Where is MULTILIB_ABI_FLAG defined?

Comment (mgorny) [inline media-libs/libass/libass-0.13.0.ebuild]: multilib-build.eclass

**Comment by luke-jr** [comment]
Looks good to me (but did not test).

**Comment by luke-jr** [comment]
FWIW, it was simple to fix the build on x32:

    https://github.com/libass/libass/pull/200

    https://github.com/libass/libass/pull/200.patch

This patch should apply cleanly to all versions in the portage tree.

**Comment by mgorny** [comment]
@bjacquin, I guess Luke's patch would be better. I would wait a while what upstream says, and possibly switch to that.

**Comment by bjacquin** [comment]
ack, less patch I can maintain, better I feel

**Comment by astiob** [comment]
Hi, I’m from upstream. Luke’s patch is now in our git master (libass/libass@4778001b673c73133d7d1dd48a385d0ca424ccfc), but it can take a while before we make the next release. Is this a serious problem for you, or could you apply the patch on your end for now?

**Comment by mgorny** [comment]
@astiob, that's no urgent issue, we can apply the patch ourselves.

@bjacquin, fetch https://github.com/libass/libass/commit/4778001b673c73133d7d1dd48a385d0ca424ccfc.patch and update your PR, please ;-).

**Comment by bjacquin** [comment]
Done all, works OK for me

**Comment by mgorny** [comment]
Thanks a lot!

**Comment by bjacquin** [comment]
Thanks to you !

**Pull request merged**